src/test/java/org/folio/marccat/resources/AuthorityRecordTest.java

```diff
@@ -22,15 +25,18 @@
 @ActiveProfiles("test")
 public class AuthorityRecordTest extends TestBase {
 
+  private static final String CONTENT_TYPE = "Content-Type";
+  private static final String FILE_TYPE = "application/json";
```

Comment:
It is preferable to move these common properties

Reply:
I have moved them.